rviz/debug.rviz

```diff
@@ -18,7 +19,7 @@ Panels:
       - /2D Nav Goal1
       - /Publish Point1
     Name: Tool Properties
-    Splitter Ratio: 0.588679
+    Splitter Ratio: 0.588679016
```

Comment:
Minor changes in `rviz/debug.rviz` should be removed (e.g. replacing 0.588679 by 0.588679016 etc). Only major changes should be included in the commit (like addition of a new display panel in RViz).